Owncloud iOs Client/Tabs/FileTab/Share/ShareLinkViewController.m

```diff
@@ -213,7 +354,11 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
 
 - (CGFloat)tableView:(UITableView *)tableView heightForFooterInSection:(NSInteger)section
 {
-    return heightOfShareLinkOptionFooterSection;
+	if ([ShareUtils hasOptionLinkNameToBeShown] && section == 0) {
```

Comment:
Here instead of comparing the section with 0, you can take advantage of NS_ENUM and make the sections to be NSUInteger. This way you can have more clean comparisons i.e.
```objective-c
if ([ShareUtils hasOptionLinkNameToBeShown] && section == LinkSectionName) {...}
```

Reply:
Yes, absolutely the enum are for that 👍